docs/07. media library/03. console commands.md

```diff
@@ -0,0 +1,22 @@
+# Console commands
+
+## Delete frontend cache of the generated thumbnails
+
+```bash
+app/console media_library:cache:clear
+```
+
+Will delete the `/src/Frontend/Files/Cache/media_library_*` folders. Which means that all frontend thumbnails will be generated next time on load.
+
+By default the backend thumbnails will remain cached. If you also want to remove the generated backend thumbnails you need to append the ` --all` option.
+
+
+## Delete all MediaItem entities
+
+```bash
+app/console media_library:delete:items
```

Comment:
Like I said, the method indeed deletes "unused items", but if you throw an extra argument at it, it deletes "all items". So renaming to delete:unused is wrong.